application/views/todolist.php

```diff
@@ -0,0 +1,42 @@
+<div class="container">
+    <h1 class="lead text-center">TODO LIST EXERCISE</h1>
+    <div class="row">
+        <div class="col-sm-6">
+            <div class="card">
+                <div class="card-header lead text-bold text-lg text-center">
+                    CREATE TO DO LIST
+                </div>
+                <div class="card-body">
+                    <button type="button" class="btn btn-primary btn-block btn-sm" id="thatsme">
+                        ADD ITEM
+                    </button>
+                </div>
+            </div>
+        </div>
+        <div class="col-sm-6">
+        </div>
+    </div>
+</div>
+
+<!-- Modal -->
+<div class="modal fade" id="create_list" data-backdrop="static" data-keyboard="false" tabindex="-1" aria-labelledby="staticBackdropLabel" aria-hidden="true">
```

Comment:
you can remove this now since you've already created a separate file for modal content.